CHANGELOG.md

```diff
@@ -370,7 +371,7 @@ To activate it set:
 
 ```js
 Sentry.config("___DSN___", {
-  deactivateStacktraceMerging: false,
```

Comment:
Please try to avoid these unrelated changes to the PR. This way the reviewers can focus on what really matters here.
Feel free to open a new PR for any styling or refactor changes.